sympy/parsing/mathematica.py

```diff
@@ -428,3 +428,20 @@ def parse(self, s):
 #        s = cls._replace(s, '}')
 
         return s
+
+if __name__ == "__main__":
+    testlist = [
+        'Sqrt[x]',
```

Comment:
Remove this whole `__main__` block from here. Include tests within `test_mathematica.py` of parsing/tests.